microprofile.cpp

```diff
@@ -4102,16 +4136,16 @@ void MicroProfileDumpHtml(MicroProfileWriteCallback CB, void* Handle, uint64_t n
 			pColorStr = &ColorString[0];
 
 		}
-		MicroProfilePrintf(CB, Handle, "S.GroupInfo[%d] = MakeGroup(%d, \"%s\", %d, %d, %d, %f, %f, %f, '%s');\n", 
+		MicroProfilePrintf(CB, Handle, "S.GroupInfo[%d] = MakeGroup(%d, \"%s\", %d, %d, %d, %s, %s, %s, '%s');\n", 
```

Comment:
this is an example of where it breaks.

Reply:
You are right. Will try to find out, I have several toughts how to do this in simple way.